app/models/feedback.rb

```diff
@@ -69,8 +70,17 @@ def send_report
       username: username,
       user_agent: user_agent,
       site_revision: ArchiveConfig.REVISION.to_s,
-      rollout: rollout
+      rollout: rollout,
+      ip_address: ip_address,
+      referer: referer,
```

Comment:
Did we get a definite answer from Support on how many characters were allowed in this field to make sure we avoid a repeat of [AO3-6692](https://otwarchive.atlassian.net/issues/AO3-6692)?

Reply:
They asked what PAC uses, and PAC said 255. I've added something to truncate that just in case, since we technically support something silly like 20k.

Reply:
Makes sense! If Support has something longer than 255, passing it to PAC will probably not work super great.